src/uct/ib/rc/base/rc_iface.c

```diff
@@ -641,20 +643,13 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_rc_iface_ops_t *ops, uct_md_h md,
                                                config->super.rx.queue_len);
         self->config.fc_hard_thresh  = ucs_max((int)(self->config.fc_wnd_size *
                                                config->fc.hard_thresh), 1);
-
-        /* Create mempool for pending requests for FC grant */
-        status = ucs_mpool_init(&self->tx.pending_mp,
-                                0,
-                                init_attr->fc_req_size,
-                                0,
-                                1,
-                                128,
-                                UINT_MAX,
-                                &uct_rc_fc_pending_mpool_ops,
-                                "pending-fc-grants-only");
+        /* Create mempool for pending requests */
+        status = uct_rc_iface_init_pending_mp(self);
         if (status != UCS_OK) {
             goto err_cleanup_rx;
         }
+
+        self->tx.pending_mp_initialized = 1;
```

Comment:
no need, it is set inside `uct_rc_iface_init_pending_mp`

Reply:
ok, removed

Reply:
unhandled